javascriptv3/example_code/bedrock-runtime/tests/converse.integration.test.js

```diff
@@ -0,0 +1,32 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { describe, expect, test } from "vitest";
+import { execSync } from "child_process";
+import path from "path";
+
+describe("Converse with text generation models", () => {
+  const file = "converse.js";
+
+  const subdirectories = [
+    "ai21LabsJurassic2",
+    "amazonTitanText",
+    "anthropicClaude",
+    "cohereCommand",
+    "metaLlama",
+    "mistral",
+  ];
+
+  const baseDirectory = path.join(__dirname, "..", "models");
+
+  test.each(subdirectories)(
+    "should invoke the model and return text",
+    (subdirectory) => {
+      const script = path.join(baseDirectory, subdirectory, file);
+      const output = execSync(`node ${script}`, {
```

Comment:
This is almost certain to cause a fake SEV-2. Our scanners don't take kindly to executing arbitrary code. In addition, you could let this run on import and do something like
```
await import("path/to/script")
```

Even better would be a similar implementation to [this file](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javascriptv3/example_code/dynamodb/actions/put-item.js) so you can control when things are executed.

Reply:
@cpyle0819 makes an excellent point, and I think the .NET base test may have a similar problem. Could we be as specific as possible about what's being run?